manifests/params.pp

```diff
@@ -15,7 +15,7 @@
   $package_name               = 'NSClient++ (x64)'
   $package_source             = 'NSCP-0.5.1.28-x64.msi'
   $download_destination       = 'c:/temp'
-  $config_template            = 'nsclient/nsclient.ini.erb'
+  $config_template            = 'nsclient/nsclient.ini.epp'
```

Comment:
so the old erb template in the repository can be deleted?

Reply:
yes. removed.

Reply:
yes, now removed